uxarray/grid/grid.py

```diff
@@ -897,13 +1072,85 @@ def bounds(self):
 
         return self._ds["bounds"]
 
+    @bounds.setter
+    def bounds(self, value):
+        """Setter for ``bounds``"""
+        assert isinstance(value, xr.DataArray)
+        self._ds["bounds"] = value
+
     @property
     def face_jacobian(self):
         """Declare face_jacobian as a property."""
         if self._face_jacobian is None:
             _ = self.face_areas
         return self._face_jacobian
 
+    def chunk(self, n_node="auto", n_edge="auto", n_face="auto"):
```

Comment:
Should this be added to the user api somewhere?

Reply:
Good catch, forgot to include this. Will do so!

Reply:
> Should this be added to the user api somewhere?

Hey @aaronzedwick can you try to run some dask/chunking benchmarks on Argonne's Chrysalis machine?

Reply:
I could, it would be a lot easier if I did that after this was released though, so I can just get the latest UXarray version. Adding an ASV benchmark might be a good idea, not sure exactly how something like this should be benchmarked though @philipc2.

Reply:
This PR won't immediately improve the performance of any of our functionality. It adds support for repressing the variables as Dask array.

Reply:
Ok, we can get to it in the next iteration.